opennlp-tools/src/main/java/opennlp/tools/namefind/NameFinderEventStream.java

```diff
@@ -136,6 +140,20 @@ public NameFinderEventStream(ObjectStream<NameSample> dataStream) {
     return generateEvents(tokens, outcomes, contextGenerator).iterator();
   }
 
+  private Span[] overrideDefaultType(Span[] names) {
+    // need to copy because Span is not modifiable
+    Span[] converted = new Span[names.length];
```

Comment:
Do we need to create a new array here? Would it make sense to do this in place in the original array?